.travis.yml

```diff
@@ -6,8 +6,7 @@ node_js:
 env:
```

Comment:
@HarshKhandeparkar @aashna27 Any ideas why this is failing on node 8?

Reply:
Can this help > https://stackoverflow.com/a/12116713 ?

Reply:
Just noting that I marked this okay since the build seems to pass in node 10, which indicates that there's no problem with the code but the module itself, and a few minor changes should make it work!

Reply:
I still think that it is a problem with JSDOM. When I initially added the tests, I found that most of the ui tests are not run. I fixed that and it worked for me locally(maybe becuase I use node 10). When I pushed the commit, it failed in node 8 but not in node 10. 

Is there a way to test it locally in node 8? Does anyone use node 8 or is there a way to run a virtual environment?

@rexagod @aashna27

#### Thanks a lot

Reply:
@HarshKhandeparkar Try using `nvm` to run different node versions on your system at a time. Also, don't forget to revert to the latest lts when you're done!

Reply:
Ok. I will try that. Thanks